test-files/cmd/glacial/DiffConflateCmdTest.sh

```diff
@@ -17,7 +17,12 @@ echo ""
 echo "Running diff..."
 echo ""
 hoot conflate $LOG_LEVEL $CONFIG $GENERAL_OPTS \
- $INPUT_DIR/input1.osm $INPUT_DIR/input2.osm $OUTPUT_DIR/output.osm --differential
+  $INPUT_DIR/input1.osm $INPUT_DIR/input2.osm $OUTPUT_DIR/output.osm --differential
```

Comment:
Script tests validate test output with this code block. Convention is to name the output validation report by appending "-validation-report" and no file extension to the base name of  the test output file being validated.